TopMovies/Service/MovieService.swift

```diff
@@ -34,10 +34,22 @@ class MovieService: StoreSubscriber {
               .completed(MovieCategory(dto: $0),
                          $0.results.map { Movie(dto: $0) })}
```

Comment:
```swift
$0.results.map(Movie(dto:))
```

Reply:
It should work theoretically, but it doesn't :)

Reply:
Movie.init(dto:)